assets/src/scripts/store/index.js

```diff
@@ -294,6 +302,12 @@ export const Actions = {
             type: 'SET_GAGE_HEIGHT',
             gageHeight
         };
+    },
+    setLocationIANATimeZone(ianaTimeZone) {
```

Comment:
I've been starting to follow best practices as described here: https://medium.com/@kylpo/redux-best-practices-eef55a20cc72. They recommend <NOUN><VERB> for action constants NOUN_VERB
.